nominees/notesmaster.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "NotesMaster",
+  "aliases": [
+    "Notesmaster Elearning",
+    "NM"
+  ],
+  "description": "NoteMaster provides training and support to use the NotesMaster elearning platform, which is preconfigured with local curricula, so the trained teachers can create digital lessons online in Notesmaster, for students to then access. The lessons can be accessed and remixed by teachers from any other country to suit their local curricula. These lessons are then submitted to a peer review system which leads to a sustainable multiplier effect in the development of new digital lessons.",
+  "website": "https://notesmaster.com/",
+  "license": [
+    {
+      "spdx": "CC-BY-SA-4.0",
+      "licenseURL": "https://notesmaster.com/termsofuse"
```

Comment:
I see in the conversation that the Terms of Use will be updated. However, the website still shows the October 2020 Terms. I think we should hold on this until the new website is up and ready.

Reply:
The terms of use have now been updated, please check the link:
https://notesmaster.com/termsofuse

Further, click: https://notesmaster.com/ and scroll down to see that the new homepage has been updated with the text:
Digital Content Licence. All content created on NotesMaster by its users and administrators is given the following attribution: Attribution-ShareAlike 4.0 International (CC BY-SA 4.0)